action.yml

```diff
@@ -62,6 +62,16 @@ inputs:
       description: 'Annotate the target namespace'
       required: false
       default: true
+   private-cluster:
```

Comment:
Need to also document these in README.md. Document in table of inputs.

Also add an example for private clusters.